test/utils/DateFormatter.test.ts

```diff
@@ -0,0 +1,50 @@
+import { expect } from 'chai';
+
+import { dateToLocalTimeString } from 'app/utils/DateFormatter';
+
+describe('DateFormatter', () => {
+
+    it('should throw invalid date error', () =>{
+
+        const date: Date = new Date('nonsense date value');
+        expect(() => dateToLocalTimeString(date)).throws('DateFormatter - Invalid date');
+
+    });
+
+    it('should return YYYY-MM-DD format for a local date', () =>{
+
+        const date: Date = new Date('2020-10-30T23:00:00.000');
+        expect(dateToLocalTimeString(date)).to.equal('2020-10-30');
+
+    });
+
+    it('should return correct month number for a given month value', () =>{
```

Comment:
I do not really understand what that test scenario means. What does `correct month number for a given month value` really mean?

Reply:
when you set values you can set:

- Dates to any value from 1 to 31
- Months to any value from 0 to 11. However in string format we want to make sure to add 1 to that value so that we store and display months from 1-12